Presentation/Presentation.swift

```diff
@@ -177,6 +177,11 @@ public extension Presentation {
         return new
     }
 
+    /// Returns a new presentation where `callback` will be called for every values.
```

Comment:
values -> value (and perhaps `to every signaled value`?)

Reply:
Fixed